designer/client/src/components/toolbars/scenarioActions/buttons/DeployButton.tsx

```diff
@@ -39,7 +44,7 @@ export default function DeployButton(props: ToolbarButtonProps) {
     const { open } = useWindows();
 
     const message = t("panels.actions.deploy.dialog", "Deploy scenario {{name}}", { name: processName });
-    const action = (p, c) => HttpService.deploy(p, c).finally(() => dispatch(loadProcessState(processName, processVersionId)));
+    const action = (p, c, d) => HttpService.deploy(p, c, d).finally(() => dispatch(loadProcessState(processName, processVersionId)));
```

Comment:
Can we find a better name for these (p, c, d) parameters?

Reply:
Fixed, full parameter names.